pyseext/js/PySeExt.GridHelper.js

```diff
@@ -181,5 +180,54 @@ globalThis.PySeExt.GridHelper = {
         }
 
         return columnHeader;
+    },
+    /**
```

Comment:
Need a space between the methods please.

Also, this method does not return a row record, but the index for the row.
It's name should be changed and header updated please.

Reply:
Updated accordingly